python/lsst/dax/imgserv/getimage/coadd.py

```diff
@@ -0,0 +1,177 @@
+#
+# LSST Data Management System
+# Copyright 2008, 2009, 2010, 2011, 2012 LSST Corporation.
```

Comment:
Is this a rename of a pre-existing file? If it's new it can't have a copyright owned by LSST Corp from 2012.

Reply:
It's a file recovered  from coadd_utils package, which has deleted it as part of package cleaning up per approved RFC 565. ImgServ still needs it for image stitching.